dist/images/ovnkube.sh

```diff
@@ -194,6 +195,8 @@ ovn_sb_raft_election_timer=${OVN_SB_RAFT_ELECTION_TIMER:-1000}
 ovn_hybrid_overlay_enable=${OVN_HYBRID_OVERLAY_ENABLE:-}
 ovn_hybrid_overlay_net_cidr=${OVN_HYBRID_OVERLAY_NET_CIDR:-}
 ovn_disable_snat_multiple_gws=${OVN_DISABLE_SNAT_MULTIPLE_GWS:-}
+# OVN_DISABLE_FORWARDING_ON_INTERFACES - disable forwarding on OVNK controlled interfaces
```

Comment:
redundant comment? it is already added in https://github.com/ovn-org/ovn-kubernetes/pull/3374/files#diff-ea48eab674fe05348e580125fccb19b775e0fd6f6a2d28a2dab5fb382590a16aR84